lua/ulx/modules/sh/sh_cfc_restart_ulx.lua

```diff
@@ -0,0 +1,41 @@
+require( "cfc_restart_lib" )
+local CATEGORY_NAME = "Utility"
+
+if SERVER then
+    util.AddNetworkString( "CFC_RESTART_START" )
+    util.AddNetworkString( "CFC_RESTART_STOP" )
+end
+
+local function cfcRestartTimerCallback()
+    timer.Destroy( "CFC_RESTART_TIMER" )
```

Comment:
`timer.Destroy` is deprecated ([source](https://wiki.facepunch.com/gmod/timer.Destroy)), and could be removed in the future. It's better practice to use `timer.Remove`, works in exactly the same way.
```suggestion
    timer.Remove( "CFC_RESTART_TIMER" )
```